src/main/java/study/racingcar/domain/Name.java

```diff
@@ -0,0 +1,24 @@
+package study.racingcar.domain;
+
+public class Name {
+	private static final int NAME_MAX_LENGTH = 5;
+	private static final String CAR_NAME_LENGTH_MESSAGE_FRONT = "자동차의 이름은 ";
+	private static final String CAR_NAME_LENGTH_MESSAGE_BACK = "자를 초과할 수 없습니다.";
+
+	private final String name;
+
+	public Name(String name) {
+		this.name = name;
+		checkLength();
+	}
+
+	public void checkLength() {
+		if(this.name.length() > NAME_MAX_LENGTH){
```

Comment:
띄어쓰기 확인 부탁드릴게요.
```suggestion
                if (this.name.length() > NAME_MAX_LENGTH) {
```
그리고 자동차 이름이 공백(`""`)인 경우는 어떻게 해야 할까요? 🤔